README.md

```diff
@@ -6,6 +6,7 @@ https://github.com/seccomp/libseccomp
 [![Build Status](https://github.com/seccomp/libseccomp/workflows/Continuous%20Integration/badge.svg?branch=main)](https://github.com/seccomp/libseccomp/actions)
 [![Coverage Status](https://img.shields.io/coveralls/github/seccomp/libseccomp/main.svg)](https://coveralls.io/github/seccomp/libseccomp?branch=main)
 [![Language grade: C/C++](https://img.shields.io/lgtm/grade/cpp/g/seccomp/libseccomp.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/seccomp/libseccomp/context:cpp)
++[![Doxygen Documentation](https://img.shields.io/badge/Documentation-Doxygen-green.svg)](https://seccomp.github.io/libseccomp/)
```

Comment:
Is it worth putting the docs under a "doxygen" dir, e.g. https://seccomp.github.io/libseccomp/doxygen, in case we ever want to have a proper seccomp/libseccomp landing page that doesn't immediately send one to the doxygen docs?

As a related FYI, I've had the libseccomp.io domain for a while now but haven't really used it anywhere; it just redirects to github.com/seccomp right now.

Reply:
Yeah, I think that's a good idea.  As I mentioned above, we may want to have doxygen for different branches as well.  Using a subdir would easily allow for multiple doxygens and possibly other HTML stuff

Reply:
Based on this discussion, I made two changes:
1.  The destination github branch is now called `html`.  (Previously I called it `doxygen/main`)
2. The doxygen output is now published to a subdir called `doxygen/main`

Currently, I'm restricting the doxygen logic to only run on the `main` branch, but this should allow us to publish doxygen for release branches as well.  Also, it will allow the creation of other HTML info if/when we ever generate it.

Reply:
Works for me, thanks!